detection_rules/main.py

```diff
@@ -155,17 +153,56 @@ def mass_update(ctx, query, metadata, language, field):
 
 
 @root.command('view-rule')
-@click.argument('rule-file')
+@click.argument('rule-file', type=Path)
 @click.option('--api-format/--rule-format', default=True, help='Print the rule in final api or rule format')
 @click.pass_context
-def view_rule(ctx, rule_file, api_format):
+def view_rule(ctx, rule_file, api_format, verbose=True):
```

Comment:
calling this function without `verbose` is a no-op.
i would leave it.

if you want to get the pretty format as a string, then do that.
can add `TOMLRule.as_toml`  and `TOMLRule.as_api_json` if you want